internal/scorecard/plugins/plugin_runner.go

```diff
@@ -290,6 +290,8 @@ func RunInternalPlugin(pluginType PluginType, config BasicAndOLMPluginConfig, lo
 				Client:   runtimeClient,
 				CR:       obj,
 				ProxyPod: proxyPodGlobal,
+				Selector: config.Selector,
+				Version:  config.Version,
 			}
 			basicTests := NewBasicTestSuite(conf)
```

Comment:
Rather than passing `config.Selector` and `config.Version` down into the specfic test suite configs, WDYT about implementing something like `func (ts *TestSuite) ApplySelector(sel labels.Selector)` that generically filters tests out of the suite, and then call it right after constructing the suite.

I'm guessing this would make the `--list` implementation more straightforward as well.

Reply:
I'll give this a try.

Reply:
ok, new implementation, better than before, good suggestion Joe.